script/openqa-clone-custom-git-refspec

```diff
@@ -157,6 +182,12 @@ Please try 'curl $json_url' or select another job, e.g. in the same scenario: $h
     local scriptdir
     scriptdir=$(dirname "${BASH_SOURCE[0]}")
     local cmd="$dry_run $scriptdir/openqa-clone-job $clone_args \"$host\" \"$job\" _GROUP=\"$GROUP\" TEST+=\"$test_suffix\" BUILD=\"$build\" CASEDIR=\"$casedir\" PRODUCTDIR=\"$productdir\" NEEDLES_DIR=\"$needles_dir\""
+    if [[ $cmd =~ (SCHEDULE|YAML_SCHEDULE) ]]; then
```

Comment:
is this an and? or an or?

Reply:
it's an or, my bad. Though changing it to
```bash
if [[ $cmd =~ SCHEDULE ]] && [[ $cmd =~ YAML_SCHEDULE ]]; then
...
```
doesnt seem to change anythin rn so my problem may be that I am using `cmd`?